calc/src/mechanics/gen789.ts

```diff
@@ -2747,9 +2746,9 @@ export function calculateAtModsSMSSSV(
 
   // Fields - Attack Modifiers
 
-  // Haunted Graveyard - Dazzling Gleam, Draining Kiss, Foul Play, and Spirit Break deal 1.2x damage
-  if (field.chromaticField === 'Haunted-Graveyard' && move.named('Dazzling Gleam', 'Draining Kiss', 'Foul Play', 'Spirit Break')) {
-    atMods.push(4915);
+  // Forgotten Battlefield - Gigaton Hammer deals neutral Double damage to Fighting and steel types
+  if (field.chromaticField === 'Forgotten Battlefield' && move.named('Gigaton Hammer') && (defender.hasType('Steel') || defender.hasType('Fighting'))) {
+    atMods.push(8192);
```

Comment:
polish: small text changes
```suggestion
  // Forgotten Battlefield - Gigaton Hammer deals double damage to Fighting and Steel types
  if (field.chromaticField === 'Forgotten Battlefield' && move.named('Gigaton Hammer') && (defender.hasType('Steel') || defender.hasType('Fighting'))) {
    atMods.push(8192);
```

Reply:
done good spot